targets/standard.props

```diff
@@ -15,8 +15,6 @@
     <EnvironmentVariables Include="LatestCommit=$(LatestCommit)" />
     <EnvironmentVariables Include="OfficialBuildId=$(OfficialBuildId)" />
     <EnvironmentVariables Include="VersionSeedDate=$(OfficialBuildId)" />
-    <EnvironmentVariables Include="OverridePackageSource=$(SourceBuiltPackagesPath)" />
-    <EnvironmentVariables Include="OverridePackageSourceEx=$(PrebuiltPackagesPath)" Condition="'$(OfflineBuild)' == 'true'" />
-    <EnvironmentVariables Include="ExcludeInternetFeeds=true" Condition="'$(OfflineBuild)' == 'true'" />
+    <EnvironmentVariables Include="DotNetSourceBuildRestoreSources=$(SourceBuiltPackagesPath)" />
```

Comment:
I think you should still support the OfflineBuild scenario.  Basically add `$(PrebuiltPackagesPath)` to `DotNetSourceBuildRestoreSources` if it is true.  And also set `$(DotNetBuildOffline)` when `$(OfflineBuild)` is true.

Reply:
Yeah I was thinking about that but wasn't sure if the semacolon delimiter would make it through to the env variable. 

As for the offline build my assumption is we would set it globally, although I can do that with this PR.